packages/@ngtools/logger/src/null-logger.ts

```diff
@@ -1,13 +1,10 @@
-import {Logger} from './logger';
-
-import {Observable} from 'rxjs/Observable';
-
-import 'rxjs/add/observable/empty';
+import { EmptyObservable } from 'rxjs/Observable/EmptyObservable';
```

Comment:
Lower case 'o' in `rxjs/observable/EmptyObservable`.
Also rxjs v6 appears to be moving towards using creation helpers over class instantiation.  So may want to go with `import { empty } from 'rxjs/observable/empty';`